test/Frontend/crash.swift

```diff
@@ -16,6 +16,10 @@
 // CHECK-ALLOW: Program arguments: {{.*}} -experimental-allow-module-with-compiler-errors
 // CHECK-ALLOW: Compiling with effective version
 
+// RUN: not --crash %target-swift-frontend -typecheck -debug-crash-after-parse -experimental-allow-module-with-compiler-errors -swift-version 5 %s 2>&1 | %FileCheck -check-prefix CHECK-CURRENT %s
```

Comment:
This is a little annoying since we'll have to update this when the major version changes, but unfortunately `-swift-version 4` is included in the test options. Do you have any better ideas here @xymus?

Reply:
Hum, yeah, that automatic `-swift-version 4` is unfortunate. We don't update the language version that often, I think this is fine.